### PR TITLE
Add Japanese Baseline

### PR DESCRIPTION
**Pull request by xutaima**

### CODE REVIEW

File: _pages/2021/shared-tasks/simultaneous.md [hunk 1]
```diff
@@ -108,6 +109,9 @@ If you encounter a bus error similar to this [issue](https://github.com/pytorch/
 
 When submitting your system, please make sure it works for the MuST-C dev and test sets. During the official evaluation, we will run the submitted system with the blind set.
 
+### English-to-Japanese Text-to-Text Translation
+You can find instructions to train and evaluate an English-to-Japanese baseline system [here](https://github.com/pytorch/fairseq/blob/master/examples/simultaneous).
```
Comment (jmp84) [inline _pages/2021/shared-tasks/simultaneous.md]: Can you fix the link:

```
https://github.com/pytorch/fairseq/blob/master/examples/simultaneous_translation/docs/enja-waitk.md
```

**Review by jmp84 — COMMENTED** [review]
Thanks for the new baseline! Please fix the link.

File: _pages/2021/shared-tasks/simultaneous.md (hunk 1 above)
Comment (xutaima) [reply _pages/2021/shared-tasks/simultaneous.md]: Thank you so much! It's fixed

**Review by jmp84 — APPROVED** [review]
LGTM!

**Pull request merged**